adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -42,7 +42,7 @@ import org.bdgenomics.adam.converters._
 import org.bdgenomics.adam.instrumentation.Timers._
 import org.bdgenomics.adam.io._
 import org.bdgenomics.adam.models._
-import org.bdgenomics.adam.projections.{ AlignmentRecordField, NucleotideContigFragmentField, Projection }
+import org.bdgenomics.adam.projections.{ FeatureField, AlignmentRecordField, NucleotideContigFragmentField, Projection }
```

Comment:
Please keep all imports alphabetically sorted (FeatureField should be between AlignmentRecordField/NCFField) and break this line as it is too long.